File: Section1 - Basics/Patterns/pattern12.js
function printTriangle(n) {
	let k = 2 * n - 2;
	let str = "";
	for (let i = 1; i <= n; i++) {
		for (let j = 1; j <= i; j++) {
			str = str + j + " ";
		}
		if (k > 0) {
			str = str + " ".repeat(2 * k);
		}
		for (let x = i; x > 0; x--) {
			str = str + x + " ";
		}
		k = k - 2;
		str += "\n";
	}
	console.log(str);
}

printTriangle(5);

File: Section1 - Basics/PatternsJava/pattern12.java
void printTriangle(int n) {
    // code here
    int k=2*n -2;
    for(int i=1; i<=n; i++) {
        for(int j=1; j<=i; j++) {
            System.out.print(j + " ");
        }
        if(k>0) {
            System.out.print(" ".repeat(2*k));
        }
        for(int j=i; j>0; j--) {
            System.out.print(j+ " ");
        }
        k=k-2;
        System.out.print("\n");
    }
}